components/tekton/pipelines/run-vse-sync-testsuite/run-vse-sync-testsuite.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: run-vse-sync-testsuite
+spec:
+  finally:
+    - name: send-report-to-gchat-space
+      params:
+        - name: webhook-url
+          value: >-
+            https://chat.googleapis.com/v1/spaces/AAAAiCQuo38/messages?key=AIzaSyDdI0hCZtE6vySjMm-WEfRq3CPzqKqqsHI&token=QjLrTKvtNWmxFlkce7kVXJu6pzzlLtACBo8I4xem-AU%3D
+        - name: success-message
+          value: All Tests passed
+        - name: failure-message
+          value: The following tests failed
+      taskRef:
+        kind: Task
+        name: send-report-to-gchat-space
```

Comment:
Maybe you can use the [`sent-to-gchat`](https://github.com/redhat-partner-solutions/vse-catalog/blob/main/components/tekton/tasks/send-to-gchat.yaml) task we have in the catalog?

Reply:
I made some modification to send-to-gchat to report the results I thought it should be distinct as its not just a simple message. [report version](https://github.com/redhat-partner-solutions/vse-catalog/pull/28/files#diff-27207940c4b29e0968c81a332c02586927f20370a4a5d7922f8735810936d6d8R32)